### PR TITLE
Add post, search and status apis for request for help feature

### PR DESCRIPTION
**Pull request by rtkanan**

### CODE REVIEW

File: models/RequestModel.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const RequestModel = new Schema(
+  {
+    act: { type: String, required: false },
```
Comment (Kailash-Sankar) [inline models/RequestModel.js:7]: act should be required, we'll set in via page always

File: models/RequestModel.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    name: { type: String, required: true },
+    mobile: { type: String, require: true },
+
+    address: { type: String, required: false },
+    area: { type: String, required: true },
```
Comment (Kailash-Sankar) [inline models/RequestModel.js:12]: should area be required, in front end forms it's kept as optional

Comment (rtkanan) [reply models/RequestModel.js:12]: In "Request for Help", I guess, we need to know in which area the help is required. That's why I made it as mandatory.

File: models/RequestModel.js (hunk 1 above)
Comment (rtkanan) [reply models/RequestModel.js]: Made the changes.

**Pull request merged**